### ISSUE TITLE
Multiple installations of package when using poetry-dynamic-versioning with path install

### ISSUE THREAD

**Issue by fredrikaverpil** [mtkennerly/poetry-dynamic-versioning#42]
Hi!

Let's say I have two projects, "a" and "b";

pyproject.toml:

```toml
[tool.poetry]
name = "a"
version = "0.0.0"

[tool.poetry.dependencies]
b = "*"

[tool.poetry-dynamic-versioning]
enable = true
vcs = "git"
pattern  = "^(?P<base>\\d+\\.\\d+\\.\\d+)(-?((?P<stage>[a-zA-Z]+)\\.?(?P<revision>\\d+)?))?"
format-jinja = """
    {%- if distance == 0 -%}
        {{- base -}}
    {%- else -%}
        {{- bump_version(base, 2) }}.dev{{ distance - 1 }}+g{{ commit }}.d{{ datetime.now().strftime('%Y%m%d%H%M%S') -}}
    {%- endif -%}
"""
format-jinja-imports = [
    { module = "datetime", item = "datetime" }
]

```

Then let's say I install the "b" package. All is nice and works fine.

But if I need to do some local development, and temporarily change the toml into the following;

```toml
[tool.poetry.dependencies]
b = { path = "/home/fredrik/code/repos/b", develop = true }
```
...followed by a `poetry install`... there will be a _new_ package installed in the site-packages of my venv. And for each time I re-run `poetry install` yet another package of "b" is installed. Basically, the installations cause accumulation of package installations and it seems Python is unable to pick the one I expect it to pick.

I no longer have a screenshot of what it looks like in the site-packages folder, but basically there were multiple package installations of "b" with the datetime string at the end of the folder name. Maybe that is what is causing this to happen (each installation folder has a unique datetime suffix).

I've also seen this accumulation of older versions of a package (using poetry-dynamic-versioning) in venvs created with `virtualenv` (not `venv`) in our CI, but that is not using the "path installation" and seem to work fine.

This is all on the latest poetry 1.1.5 with poetry-dynamic-versioning 0.12.4.

Do you think there's something we could do about this?

**Comment by fredrikaverpil** [comment]
Yes, both projects use the same jinja format.

Ok, I will file an issue and link it here.

**Comment by fredrikaverpil** [comment]
Indeed, this is a known problem with an open PR: https://github.com/python-poetry/poetry/issues/2918

**Comment by fredrikaverpil** [comment]
I think we can close this. It's a poetry problem. Seems like a fix will be implemented in https://github.com/python-poetry/poetry/pull/3900 and which I am guessing will be included in poetry 1.2.0...?
